docs/layerzero-oft-api-reference.md

```
@@ -0,0 +1,259 @@
+| `token()` | `address(this)` | underlying ERC-20 address |
+| `approvalRequired()` | `false` | `true` |
+| On send | **Burns** tokens | **Locks** tokens into adapter |
+| On receive | **Mints** tokens | **Unlocks** tokens to recipient |
+
+---
+
+## Rebalancing Call Flows
+
+### Katana → Ethereum (send vbUSDC back)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)